administration/src/bp-modules/applications/ApplicationsOverview.tsx

```diff
@@ -112,13 +137,16 @@ const ApplicationView: FunctionComponent<{ application: Application; gotDeleted:
               marginTop: '20px',
             }}>
             {height > COLLAPSED_HEIGHT ? (
-              <Button onClick={() => setCollapsed(true)} icon='caret-up'>
+              <PrintAwareButton onClick={() => setCollapsed(true)} icon='caret-up'>
                 Weniger anzeigen
-              </Button>
+              </PrintAwareButton>
             ) : null}
-            <Button onClick={() => setDeleteDialogOpen(true)} intent='danger' icon='trash'>
+            <PrintAwareButton onClick={() => printApplicationById(id)} intent='primary' icon='print'>
+              PDF Export
```

Comment:
hm since there will be a Excel export i think we should keep `pdf exportieren` They want to export and save the document and it may be confusing it it say `Drucken`